examples/misc/lib/effective_dart/usage_bad.dart

```diff
@@ -152,6 +139,34 @@ void miscDeclAnalyzedButNotTested() {
 
 //----------------------------------------------------------------------------
 
+class Player {
+  bool get isActive => false;
+}
```

Comment:
You can drop `class Player` since `usage_good.dart` is imported and it defines `Player`.